cpp/src/community/detail/refine_impl.cuh

```diff
@@ -230,6 +229,7 @@ refine_clustering(
     cugraph::reduce_op::plus<weight_t>{},
     weighted_cut_of_vertices_to_louvain.begin());
 
+  // FIXME: Consider using bit mask logic here.  Would reduce memory by 8x
```

Comment:
Yes, and also a higher chance to fit in L2 cache.

Reply:
We have multiple utility functions to support this.

https://github.com/rapidsai/cugraph/blob/branch-24.12/cpp/include/cugraph/utilities/packed_bool_utils.hpp